vendor/modules.txt

```diff
@@ -1179,6 +1179,10 @@ k8s.io/component-base/logs
 k8s.io/component-base/logs/api/v1
 k8s.io/component-base/metrics
 k8s.io/component-base/metrics/legacyregistry
+k8s.io/component-base/metrics/prometheus/clientgo
```

Comment:
(not related to this line)

Could you run `go mod tidy && go mod vendor` locally and submit the vendor/ changes as well?

Reply:
Sure.